docs/src/componentDocs/DrawerBody/examples/BasicDrawerBody.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { CodeBlock, CopyToClipboard, FullCodeOnGithub } from '../../../shared';
+import { BasicDrawerBodyExample } from './BasicDrawerBodyExample';
+
+const codeSnippet = `<Drawer open={true} width={250} sx={{ margin: '0 auto' }} noLayout>
+    <DrawerBody sx={{ flex: '1 1 auto' }}>
```

Comment:
We are adding this `noLayout`, and `sx` prop only for the Example page. Since all drawer examples are not inside <DrawerLayout/>. Confirm with @joebochill  that we don't need `noLayout` prop on Drawer and `sx` prop <DrawerBody/> on from the code snippet. Since the user will use this code inside the <DrawerLayout/> component. 

@bkarambe, @daileytj. Let me know if we need to discuss this to be on the same page.